assets/Button.scss

```diff
@@ -0,0 +1,28 @@
+@import 'assets/theme/colors';
+@import '@material/button/mdc-button';
+@import '@material/button/mixins';
+
+.mason-button {
+  @include mdc-button-base_;
+  @include mdc-button-corner-radius(2px);
+  @include mdc-button-container-fill-color(transparent);
+  @include mdc-button-ink-color(primary);
+  @include mdc-states(primary);
+  @include mdc-button--stroked_;
+  @include mdc-button-stroke-width(1px);
+  @include mdc-button-stroke-color(primary);
+  @include mdc-button__icon-contained_;
+
+  // The icon CSS class overrides styles defined in the .material-icons CSS
+  // class, which is loaded separately so the order of CSS definitions is not
+  // guaranteed. Therefore, increase specifity by nesting this class to ensure
+  // overrides apply.
+  &__icon {
```

Comment:
The `&` in SCSS will convert to the parent class so this is compiled to `.mason-button__icon`